File: JAVA/Calculator.java
package Calculator;

import java.util.Scanner;

public class Calculator {
    
    public static void main(String[] args) {
        Scanner getUserInput = new Scanner(System.in);
        
        System.out.print("Enter your First Number: ");
        if (!getUserInput.hasNextDouble()){
            System.out.println("Invalid Input! Must be a Number!");
            getUserInput.close();
            return;
        }
        
        double firstNumber = getUserInput.nextDouble();
        
        System.out.println(" ( +, -, *, / ) ");
        System.out.print("Enter your Operation: ");
        char operations = getUserInput.next().charAt(0);
        
        System.out.print("Enter your Second Number: ");
        if (!getUserInput.hasNextDouble()){
            System.out.println("Invalid Input! Must be a Number!");
            getUserInput.close();
            return;
        }
        
        double secondNumber = getUserInput.nextDouble();
        
        double result;
        
        switch (operations){
            case '+':
            result = firstNumber + secondNumber;
            break;
            
            case '-':
            result = firstNumber - secondNumber;
            break;
            
            case '*':
            result = firstNumber * secondNumber;
            break;
            
            case '/':
            if (secondNumber == 0){
                System.out.println("Cannot be divided into zero");
                getUserInput.close();
                return;
            }
            result = firstNumber / secondNumber;
            break;
            
            default:
            System.out.println("Invalid Operation");
            getUserInput.close();
            return;
            
        }
        
        System.out.println("Result: " + result);
        getUserInput.close();
           
    }
    
}

File: C/calculator.c
#include <stdio.h>

int main(){
   
   float result = 0;
   float a = 0;
   float b = 0;
   char c = '\0';
   
   printf("Enter the first number: ");
   scanf("%f", &a);
   
   printf("Enter the second number: ");
   scanf("%f", &b);
   
   printf("Enter an operator (+ - * /): ");
   scanf(" %c", &c);
   
   switch(c){
      case '+':
      result = a + b;
      printf("Sum: %.2f", result);
      break;
      
      case '-':
      result = a - b;
      printf("Difference: %.2f", result);
      break;
      
      case '*':
      result = a * b;
      printf("Product: %.2f", result);
      break;
      
      case '/':
      if(a == 0 || b == 0){
         printf("Can't divide by zero.");
      }else{
         result = a / b;
         printf("Quotient: %.2f", result);
      }
      default:
      printf("Error");
   }
}